api/v1/cluster_types.go

```diff
@@ -135,6 +135,12 @@ type ClusterSpec struct {
 	// +optional
 	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
 
+	// Name for custom sheduler when the default one cannot be used
+	// If not defined, it defaults to `default-scheduler`.
```

Comment:
I think it would be better to default to "", as that would be the same behavior we currently have and doesn't need any assumption to be made

Reply:
Okay, that makes sense.
I added additional parameter to kubebuilder:
`// +kubebuilder:default:=none`

From what I understand, it should generate by default the target result of an empty string.

Reply:
yeah, either that or just not setting a default at all would have the same effect 👍